migration-util/src/main/java/edu/unc/lib/dcr/migration/content/ContentObjectTransformer.java

```diff
@@ -381,18 +382,20 @@ private void transformPremis(PID bxc3Pid, PID bxc5Pid) {
         premisTransformer.compute();
 
         // Add migration event
-        addMigrationEvent(premisLogger);
+        addMigrationEvent(premisLogger, depositBxc3.getFullname());
     }
 
     private PremisLogger getPremisLogger(PID bxc5Pid) {
         Path transformedPremisPath = directoryManager.getPremisPath(bxc5Pid);
         return premisLoggerFactory.createPremisLogger(bxc5Pid, transformedPremisPath.toFile());
     }
 
-    private void addMigrationEvent(PremisLogger premisLogger) {
+    private void addMigrationEvent(PremisLogger premisLogger, String version) {
+        SoftwareAgentConstants.setCdrVersion(version);
```

Comment:
I think you can remove this. It looks like `TransformContentCommand` uses `service-context.xml` now as well, so cdrVersion should be getting set that way already.